supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/action/ActionUtils.scala

```diff
@@ -10,7 +10,7 @@ object ActionUtils {
   val sshEntryPoint = "ssh-server"
 
   /** Port mappings for the ssh container. */
-  val sshPortMappings = Map("22" -> new Integer(22))
+  val sshPortMappings = Map("22" -> Int.box(22))
```

Comment:
The compiler was complaining that `new Integer(n)` is deprecated. Hence updated to a more 'scala-esque' option as suggested [here](https://github.com/broadinstitute/cromwell/pull/6003#discussion_r515270682).